apps/content/migrations/0043_remove_assetaccessrequest_admin_response_and_more.py

```
@@ -0,0 +1,35 @@
+        ),
+        migrations.AddField(
+            model_name='assetaccessrequest',
+            name='rejected_by',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='rejected_asset_requests', to=settings.AUTH_USER_MODEL),
+        ),
+        migrations.AddField(
+            model_name='assetaccessrequest',
+            name='rejection_reason',
+            field=models.TextField(blank=True, help_text='Reason shown to the developer when a request is rejected'),
+        ),
+    ]
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)